saleor/order/calculations.py

```diff
@@ -200,18 +189,42 @@ def _calculate_and_add_tax(
     prices_entered_with_tax: bool,
 ):
     if tax_calculation_strategy == TaxCalculationStrategy.TAX_APP:
-        # Get the taxes calculated with plugins.
+        # If taxAppId is not configured remain previous behaviour to call plugin and apps
+        if not tax_app_identifier:
+            # Get the taxes calculated with plugins.
+            _recalculate_with_plugins(manager, order, lines, prices_entered_with_tax)
+            # Get the taxes calculated with apps and apply to order.
+            tax_data = manager.get_taxes_for_order(order, tax_app_identifier)
+            _apply_tax_data(order, lines, tax_data)
+        else:
+            _call_plugin_or_app_tax(
```

Comment:
Same sugestion as for `checkout/calculations`. I think that making something similar as in the posted example, could improve the readability in this logic (as we have a lot of if/esle conditions).